packages/core/package.json

```diff
@@ -50,6 +50,7 @@
     "datastore-level": "^8.0.0",
     "debug": "4.3.4",
     "err-code": "3.0.1",
+    "async-iterable-queue": "1.0.15",
```

Comment:
You'll probably need to push your locally modified `yarn.lock` file as well, otherwise the tests in the CI will fail instantly (because a new dep has been brought in).

Reply:
Thanks, that happened.